aio/tools/transforms/angular-base-package/post-processors/auto-link-code.js

```diff
@@ -35,9 +35,8 @@ module.exports = function autoLinkCode(getDocFromAlias) {
         }
 
         visit(node, 'text', (node, ancestors) => {
-          // Only interested in text nodes that are not inside links
-          const isInsideLink = ancestors.every(ancestor => !is(ancestor, 'a'));
-          if (!isInsideLink) {
+          const isInsideOtherLinks = isInsideLink(ancestors);
```

Comment:
```suggestion
          const isInsideOtherLink = isInsideLink(ancestors);
```

...or (to be consistent with the other call below):

```suggestion
          const isInLink = isInsideLink(ancestors);
```

Reply:
Done: https://github.com/angular/angular/pull/36686/commits/47df12e967998944ee4fb8f2515dd3d3c2c5caa3